.github/workflows/c8run-release.yaml

```diff
@@ -29,20 +38,52 @@ permissions:
   security-events: none
   statuses: write
 
+env:
+  CAMUNDA_RUN_NAME: c8run-${{ inputs.camundaVersion }}
+
 jobs:
-  release:
-    name: C8Run Release ${{ matrix.os }}
-    runs-on: ${{ matrix.os }}
+  init:
+    name: Create C8Run tag/release
+    runs-on: windows-latest
+    timeout-minutes: 15
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.branch }}
+      - name: Import Secrets
+        id: secrets
+        uses: hashicorp/vault-action@d1720f055e0635fd932a1d2a48f87a666a57906c # v3.0.0
+        with:
+          url: ${{ secrets.VAULT_ADDR }}
+          method: approle
+          roleId: ${{ secrets.VAULT_ROLE_ID }}
+          secretId: ${{ secrets.VAULT_SECRET_ID }}
+          secrets: |
+            secret/data/products/distribution/ci NEXUS_USERNAME;
+            secret/data/products/distribution/ci NEXUS_PASSWORD;
+
```

Comment:
This init job doesn't seem necessary. the windows job already calls these steps redundantly. Is there a reason for it to be here?